test/utils/devices/temperature_probe_test.py

```diff
@@ -58,23 +58,5 @@ def test_temperature_probe_resistance_null():
     """
     The function to test getting a resistance from a null mock sensor
     """
-    temperature_sensor = Temperature_Probe(None, None, None, None)
+    temperature_sensor = TemperatureProbe(None, None, None, None)
     assert temperature_sensor.get_resistance() == 1000.0
-
-
-def test_temperature_probe_set_temperature():
-    """
-    The function to test setting the temperature for the mock sensor
-    """
-    temperature_sensor = create_temperature_probe()
-    temperature_sensor.mock_set_temperature(25)
-    assert temperature_sensor.get_temperature()
-
-
-def test_temperature_probe_set_resistance():
```

Comment:
How can you achieve this test in the new code base?

Reply:
This tests the temperature probe mock set resistance function. The function has been removed as you should not be setting a resistance for the probe. The probe returns that information to you. For testing purposes of the device, we can just change the class's variable directly.